mobify/astro-scaffold
# HYB-695: Deeplinking

Enable universal deep linking for ios

JIRA: https://mobify.atlassian.net/browse/HYB-695
Linked PRs: https://github.com/mobify/generator-astro/pull/26

## Changes
- Add `continueUserActivity` in `AppDelegate`
- Add an entitlements file to build
- Add a `apple-app-site-association` which can be further configured and hosted on the deep linked site

## How to test-drive this PR
- Run the generator and ensure that the entitlements file and association file are updated correctly

## TODOS:
